frontend/packages/ux-editor/src/components/Properties/EditSubFormTableColumns/hooks/useSubFormLayoutValidation.ts

```diff
@@ -0,0 +1,15 @@
+import { useFormLayoutsQuery } from '@altinn/ux-editor/hooks/queries/useFormLayoutsQuery';
+import { getAllLayoutComponents } from '@altinn/ux-editor/utils/formLayoutUtils';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+
+export const useSubFormLayoutValidation = (subFormLayoutSetName: string): boolean => {
+  const { org, app } = useStudioEnvironmentParams();
+  const { data: formLayout } = useFormLayoutsQuery(org, app, subFormLayoutSetName);
+  var hasComponents = false;
+  if (formLayout) {
+    for (const value of Object.values(formLayout)) {
+      hasComponents ||= getAllLayoutComponents(value).length > 0;
+    }
+  }
+  return hasComponents;
```

Comment:
To avoid  using `var `and using for-loop inside if-statment, can we  just have a seperate check for existing of `formlayout `and then use `some `to find if we have a component? 
like this maybe?:
```
if (!formLayout) return false;
  return Object.values(formLayout).some(layout =>
    getAllLayoutComponents(layout).length > 0
  );
```

Reply:
Yes good suggestion! Updated the hook 👍